src/components/example/ECard.tsx

```
@@ -0,0 +1,57 @@
+                <CardTitle>Test Card</CardTitle>
+                <CardDescription>Deploy your new project in one-click.</CardDescription>
+            </CardHeader>
+            <CardContent>
+                <form>
+                    <div className="grid w-full items-center gap-4">
+                        <div className="flex flex-col space-y-1.5">
+                            <Label htmlFor="name">Name</Label>
+                            <Input id="name" placeholder="Name of your project" />
+                        </div>
+                        <div className="flex flex-col space-y-1.5">
+                            <Label htmlFor="framework">Framework</Label>
```

ESLint

[error] Replace `····························` with `··············`

(prettier/prettier)